scenarios/evaluate/evaluate_remotely/evaluate_remotely.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Remote Evaluations: Evaluating in the Cloud \n",
+    "\n",
+    "## Objective\n",
+    "\n",
+    "This tutorial provides a step-by-step guide on how to evaluate data generated by LLMs remotely in the cloud. \n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- [Azure AI Safety Evaluation](https://aka.ms/azureaistudiosafetyeval)\n",
+    "- [azure-ai-evaluation](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/develop/evaluate-sdk)\n",
+    "\n",
+    "## Time\n",
+    "\n",
+    "You should expect to spend 20 minutes running this sample. \n",
+    "\n",
+    "## About this example\n",
+    "\n",
+    "This example demonstrates the remote evaluation of query and response pairs that were generated by an LLM model. It is important to have access to AzureOpenAI credentials and an AzureAI project. **To create data to use in your own evaluation, learn more [here](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/develop/simulator-interaction-data)** . This example demonstrates: \n",
+    "\n",
+    "- Single-instance, triggered Remote Evaluation (to be used for pre-deployment evaluation of LLMs)\n",
+    "\n",
+    "## Before you begin\n",
+    "### Prerequesite\n",
+    "- [Have an online deployment on Azure Open AI studio supporting `chat completion` such as `gpt-4`](https://learn.microsoft.com/en-us/azure/machine-learning/concept-endpoints-online?view=azureml-api-2)\n",
+    "- You also might want to evaluate data generated by your LLM, to see how to generate data to be evaluated using the Azure AI Evaluation SDK, see our samples on simulation \n",
+    "\n",
+    "### Installation\n",
+    "\n",
+    "Install the following packages required to execute this notebook. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %pip uninstall azure-ai-project azure-ai-ml azure-ai-evaluation\n",
+    "# %pip install azure-identity\n",
+    "# %pip install azure-ai-project\n",
+    "# %pip install azure-ai-evaluation"
```

Comment:
bug? Installation command is commented out.

Also if the intent behind the initial uninstall was to force a user to get the latest version, you can accomplish the same with the `-U` option for pip install (i.e. `pip install -U ...`)

```suggestion
    "%pip install azure-identity\n",
    "%pip install azure-ai-project\n",
    "%pip install azure-ai-evaluation"
```